bears/ruby/RuboCopBear.py

```diff
@@ -19,7 +19,7 @@ class RuboCopBear:
     """
 
     LANGUAGES = {'Ruby'}
-    REQUIREMENTS = {GemRequirement('rubocop', '0.47.1'),
+    REQUIREMENTS = {GemRequirement('rubocop'),
```

Comment:
Our motive is to use the latest version of rubocop. We were using `0.47.1` in the earlier case and due to an upstream release i.e. `0.48`, our CI broke. As a temporary fix, I had to pin the version to `0.47.1` to make sure that the tree was green again. Now that we have to solution, I think it would be a good option to get rid of the pinning and install the latest version.